src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/UniqueAccessCodeServiceImpl.java

```diff
@@ -73,17 +87,18 @@ public UniqueAccessCodeDTO getAndAuthenticateUAC(String uacHash) throws CTPExcep
 
   /** Send UacAuthentication event */
   private void sendUacAuthenticationEvent(UniqueAccessCodeDTO data) throws CTPException {
+    UUID caseId = null;
+    if (data.getCollectionCase() != null) {
```

Comment:
i don't think this should ever be null . We are not catering for unlinked UACs now.